fiftyone/server/view.py

```
@@ -29,6 +29,17 @@
 _LABEL_TAGS = "_label_tags"
 
 
+def _make_group_field_stage(view):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_make_group_field_stage`

(ANN202)